uefi_ovmf_64/compile.sh

```diff
@@ -17,177 +17,257 @@ TARGET_ROOT="$(dirname ${PWD}/${0})"
 [ -n "$KAFL_ROOT" ] || KAFL_ROOT=${PWD}
 [ -n "$EDK2_ROOT" ] || EDK2_ROOT=$KAFL_ROOT/edk2.git
 
-[ -d $KAFL_ROOT/kAFL-Fuzzer ] || ( echo "Please set correct KAFL_ROOT" ; false )
+[ -d $KAFL_ROOT/fuzzer ] || ( echo "Please set correct KAFL_ROOT" ; false )
+
+# Release selection
 
 #BUILD=RELEASE
-BUILD=DEBUG
+#BUILD=DEBUG
+BUILD=NOOPT
+
+# Architecture selection
+
 #ARCH=IA32
 ARCH=X64
+#ARCH=3264
 TOOL=GCC5
 
-#APP=TestDecompress
 APP=TestBMP
 
-#BUILD_OPTS="-a IA32 -a X64 -b NOOPT -t CLANGSAN40 -n 8 -DDEBUG_ON_SERIAL_PORT"
-BUILD_OPTS="-a $ARCH -b $BUILD -t $TOOL -n $(nproc)"
-KAFL_OPTS="-p $(nproc) -redqueen -hammer_jmp_tables -grimoire -catch_reset"
+BUILD_OPTS=""
+OVMF_DSC_ARCH=""
+
+if test $ARCH = "X64"
+then
+  BUILD_OPTS="-a X64"
+  OVMF_DSC_ARCH="X64"
+elif test $ARCH = "IA32"
+then
+  BUILD_OPTS="-a IA32"
+  OVMF_DSC_ARCH="Ia32"
+elif test $ARCH = "3264"
+then
+  BUILD_OPTS="-a IA32 -a X64"
+  OVMF_DSC_ARCH="Ia32X64"
+else
+  echo "Bad target architecture configuration"
+  exit 1
+fi
+
+BUILD_OPTS="${BUILD_OPTS} \
+  -t ${TOOL} \
+  -b ${BUILD} \
+  -n $(nproc)"
+
+KAFL_OPTS="--redqueen --grimoire -v --debug -p 8"
 
 function install_edk2()
 {
-	# requirements on top of kAFL base install
-	sudo apt-get install nasm iasl g++ g++-multilib
-	
-	# download + apply patch unless install folder already exists
-	if [ -d $KAFL_ROOT/edk2.git ]; then
-		echo "[*] Folder exists, assume it is already patched.."
-		pushd $KAFL_ROOT/edk2.git
-	else
-		git clone https://github.com/tianocore/edk2 $KAFL_ROOT/edk2.git
-		pushd $KAFL_ROOT/edk2.git
-		git checkout -b edk2-stable201905
-		git submodule update --init --recursive
-		patch -p1 < $TARGET_ROOT/edk2_kafl.patch || exit
-	fi
-	make -C BaseTools -j $(nproc)
-	export EDK_TOOLS_PATH=$PWD/BaseTools
-	. edksetup.sh BaseTools
-	popd
+  # requirements on top of kAFL base install
+  sudo apt-get install nasm iasl g++ g++-multilib
+
+  # download + apply patch unless install folder already exists
+  if [ -d $KAFL_ROOT/edk2.git ]; then
```

Comment:
KAFL_ROOT is now the kafl tools folder. Lets EDK2_ROOT to "$WORKSPACE/edk2.git" and use it everywhere in the script?